Assets/OngSharing/IPhotonNetworking.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class IPhotonNetworking<T> : Singleton<IPhotonNetworking<T>> {
```

Comment:
We shouldn't be using Singleton. Please update this to use a `IMixedRealityNetworkingSystem` that inherits from `IMixedRealityService`. Then register it in the "Additional Service Providers Profile"